GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -13,6 +13,16 @@ namespace gdjs {
     childrenContent: { [objectName: string]: ObjectConfiguration & any };
     instances: InstanceData[];
     layers: LayerData[];
+    areaMinX: float;
```

Comment:
It's a bit surprising to have both areaMin/MaxX/Y/Z and defaultSize. Add a comment why this is the case :) (as it takes double the memory)

Reply:
Well actually not the double memory because defaultSize will be a reference to a single entity if I'm not mistaken (but we could then argue that in this case we could remove areaXxx properties?)

Reply:
These are object data. I think they are not copied for each instances.

Reply:
Ah right. Fair enough!